### PR TITLE
ConfigDropins for Bulkdata.xml is specified incorrectly for db2 and p…

### LINKED ISSUE #1939: ConfigDropins for Bulkdata.xml is specified incorrectly for db2
**Describe the bug**
ConfigDropins for Bulkdata.xml is specified incorrectly for db2 and postgres

**To Reproduce**
Try configuring the bulkdata and running a bulkdata job such as import
Check the jobinstances there should be one https://localhost:9443/ibm/api/batch/jobinstances
Restart
Check the jobinstances there should be one https://localhost:9443/ibm/api/batch/jobinstances
and it's zero

**Expected behavior**
There should be at least one after the restart.

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by prb112**
Closes #1939

Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: fhir-server/liberty-config/configDropins/defaults/bulkdata.xml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <server description="fhir-server">
-
-    <!-- Enable features -->
+    <!-- 
+        By default, in-memory derby is used for job repository, no configuration is needed.
+        If you need an alternative, check the disabled folder for you persistence datastore/provider.
```
Comment (punktilious) [inline fhir-server/liberty-config/configDropins/defaults/bulkdata.xml:4]: you => your

File: docs/src/pages/guides/FHIRServerUsersGuide.md [hunk 2]
```diff
@@ -1667,11 +1667,46 @@ The JavaBatch user is configured in `server.xml` and the `fhir-server-config.jso
...
 
-By default, in-memory Derby database is used for persistence of the JavaBatch Jobs as configured in `fhir-server/configDropins/bulkdata.xml`. Instruction is also provided in "Configuring a Liberty Datasource with API Key" section of the DB2OnCloudSetup guide to configure DB2 service in IBM Clouds as JavaBatch persistence store. The JavaBatch schema is created by default via the `fhir-persistence-schema` command line interface jar.
+By default, in-memory Derby database is used for persistence of the JavaBatch Jobs as configured in `fhir-server/configDropins/defaults/bulkdata.xml`. This database is destroyed on the restart of the IBM FHIR Server, and does not support load balancing.
 
-You can also choose to use postgresql or other RDBMS as your Job repository. To enable a postgresql job repository, uncomment the corresponding section of the `bulkdata.xml` server config.
+If one wants to support IBM Db2 on IBM Cloud, one should copy `fhir-server/configDropins/disabled/db2-cloud/bulkdata.xml` to `fhir-server/configDropins/defaults/bulkdata.xml` replacing the existing bulkdata.xml. One can configure the Datasource by setting the following environment variables:
```
Comment (lmsurpre) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:1674]: minor wording suggestion
```suggestion
To support IBM Db2 on IBM Cloud, copy `fhir-server/configDropins/disabled/db2-cloud/bulkdata.xml` to `fhir-server/configDropins/defaults/bulkdata.xml` replacing the existing bulkdata.xml. One can configure the Datasource by setting the following environment variables: 
```

File: docs/src/pages/guides/FHIRServerUsersGuide.md [hunk 3]
```diff
@@ -1667,11 +1667,46 @@ The JavaBatch user is configured in `server.xml` and the `fhir-server-config.jso
...
+| BATCH_DB_HOSTNAME | `blank`     | The hostname of the db2 instance                               |
+| BATCH_DB_NAME     | BLUDB       | The database name                                              |
+| BATCH_DB_SCHEMA   | FHIR_JBATCH | The Schema Name configured to support the Java Batch framework |
+| BATCH_DB_PORT     | 50001       | The port configured to support the database                    |
+| BATCH_DB_APIKEY   | `blank`     | The API Key for the Db2 Cloud database                         |
+
+Instruction is also provided in 'Configuring a Liberty Datasource with API Key' section of the DB2OnCloudSetup guide to configure DB2 service in IBM Clouds as JavaBatch persistence store. The JavaBatch schema is created using the `fhir-persistence-schema` command line interface jar.
+
+If one wants to support IBM Db2 with a user-name and password , one should copy `fhir-server/configDropins/disabled/db2/bulkdata.xml` to `fhir-server/configDropins/defaults/bulkdata.xml` replacing the existing bulkdata.xml. One can configure the Datasource by setting the following environment variables:
```
Comment (lmsurpre) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:1686]: minor wording suggestion
```suggestion
To support IBM Db2 with a user-name and password , copy `fhir-server/configDropins/disabled/db2/bulkdata.xml` to `fhir-server/configDropins/defaults/bulkdata.xml` replacing the existing bulkdata.xml. One can configure the Datasource by setting the following environment variables: 
```

**Review by lmsurpre — APPROVED** [review]
LGTM

**Pull request merged**